cmd/profile.go

```diff
@@ -23,7 +22,7 @@ var profilecmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(profilecmd)
-
+	profilecmd.Flags().StringVar(&profileOptions.GRPC, "gRPC", "", "use gRPC")
```

Comment:
Can we have default of 32767 port ?

Reply:
Yeah but for some reason when I try to run karmor log or profile, the default port is different for me if I don't mention `--gRPC` (when cluster is hosted in VM and accessed from outside).

Now, do we want the default scenario to be when cluster and karmor is on host itself? Then it can be default.